src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,120 @@
+import cart from "./cart.module.css";
+import prodList from "../productList/productList.module.css";
```

Comment:
use of absolute paths for better readability and understanding

Reply:
if in future i plan to move around my component folder then relative paths won't break while absolute paths will, so i chose to use relative ones and i figure that the second import can be improved, i'll just move out the common css to a common css file.